cliquet/scripts/cliquet.py

```diff
@@ -15,6 +17,12 @@ def deprecated_init(env):
 
 def init_schema(env):
     registry = env['registry']
+    settings = registry.settings
+
+    if settings.get('read_only', False):
+        message = 'You cannot migrate the database with a read_only setup.'
+        warnings.warn(message, ReadOnlyError)
```

Comment:
Yes you are totally right, also raising an exception add a traceback where I would like to just write down a message, maybe printing to stderr and exiting would be enough, what do you think?